doc/intro.rst

```diff
@@ -55,10 +55,10 @@ See the :ref:`pertinent section in the FAQ list <usingwith>`.
 Prerequisites
 -------------
 
-Sphinx needs at least **Python 2.7** or **Python 3.4** to run, as well as the
-docutils_ and Jinja2_ libraries.  Sphinx should work with docutils version 0.10
-or some (not broken) SVN trunk snapshot.  If you like to have source code
-highlighting support, you must also install the Pygments_ library.
+Sphinx needs at least **Python 3.5** to run, as well as the docutils_ and
+Jinja2_ libraries. Sphinx should work with docutils version 0.10 or some (not
+broken) SVN trunk snapshot. If you like to have source code highlighting
+support, you must also install the Pygments_ library.
```

Comment:
Now pygments has been added to requirements list. So this sentence is not needed!